backend/protzilla/utilities/utilities.py

```diff
@@ -65,7 +67,10 @@ def default_intensity_column(
     :return: The default intensity column name
     """
 
-    possible_substring_identifiers = ["intensity", "ibaq", "lfq", "spectral count"]
+    possible_substring_identifiers = [
+        *[i.value.lower() for i in IntensityType],
+        *[i.value.lower() for i in IntensityNameType],
+    ]
```

Comment:
This looks rather odd, how about 
```suggestion
    possible_substring_identifiers = [
        i.value.lower() for enum in (IntensityType, IntensityNameType) for i in enum
    ]
```

Reply:
changed